opensky/anaconda-project.yml

```diff
@@ -16,21 +16,22 @@ channels: [defaults]
 user_fields: [examples_config]
 
 packages: &pkgs
-- python=3.8
-- notebook <7
-- bokeh <2.4
-- colorcet
-- datashader =0.13.0
-- fastparquet =0.5.0
-- holoviews =1.14.5
-- matplotlib <3.5
-- pandas <1.4
-- python-snappy =0.6.0
-# Not direct deps but needed for the projects to run
-- numba <0.55
-- numpy <1.21
-- jinja2 <3
-- markupSafe =2.0.1
+- python=3.11
+- notebook<7
+- bokeh>=3.4.0
+- colorcet>=3.1.0
+- dask>=2023.11
```

Comment:
Is Dask used in this example?

Reply:
No, but adding the latest version of Dask silences some future warnings that were popping up.

Reply:
If we don't use Dask at all, let's just remove it.

Reply:
done